blueocean-web/package.json

```diff
@@ -42,15 +42,20 @@
     "window-handle": "1.0.0"
   },
   "jenkinscd": {
-    "extDependencies": [
+    "import": [
       "history",
       "immutable",
-      "react-router",
       "keymirror",
       "react-redux",
       "react-router",
       "redux",
       "redux-thunk"
+    ],
```

Comment:
Actually I was thinking about it more after I wrote the comment above ... other bundles are using most of these packages too, so I think it actually does make sense to externalize them from all bundles and `import` them everywhere (just as we're doing atm).